resolwe_bio/management/commands/generate_diffexpr.py

```diff
@@ -0,0 +1,243 @@
+        ann.save()
+
+        with open(os.path.join(self.data_dir, str(ann.id), 'stdout.txt'), 'w') as stdout:
+            stdout.write('Upload genome annotation with the generate_De django-admin command.')
+
+        self.stdout.write('Genome annotation created: {} (id={})'.format(filename, ann.id))
+
+        return ann
+
+    def generate_diffexp_data(self, group_size):
+
+        de_name = 'cuffdiff'
+        self.data_dir = settings.FLOW_EXECUTOR['DATA_DIR']
+        self.test_files_path = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), '..', '..', 'tests', 'processes', 'files'))
+        de_file = os.path.join(self.test_files_path, de_name + '.tab.gz')
+
+        self.stdout.write('---- case samples ----')
+        case_samples = self.create_expressions(group_size)
+
+        self.stdout.write('---- control samples ----')
+        control_samples = self.create_expressions(group_size)
+
+        self.stdout.write('---- upload annotation ----')
+
+        case_input = [sample.id for sample in case_samples]
+        control_input = [sample.id for sample in control_samples]
+        genome_annotation_input = self.create_genome_annotation('hg19_chr20_small.gtf.gz')
+
+        de_descriptor = {
+            'fc_threshold': 2,
+            'fdr_threshold': 0.01,
+            'case_label': 'Case group',
+            'control_label': 'Control group'
+        }
+
+        de_inputs = {
+            'case': case_input,
+            'control': control_input,
+            'annotation': genome_annotation_input.id
+        }
+
+        # Create DE data object
+        started = timezone.now()
+
+        de_obj = Data.objects.create(
+            name=self.get_name(de_name),
+            started=started,
+            finished=started + datetime.timedelta(minutes=20),
+            status=Data.STATUS_PROCESSING,
+            descriptor_schema=get_descriptorschema('diff-exp'),
+            descriptor=de_descriptor,
+            process=get_process(de_name),
+            contributor=get_superuser(),
+            input=de_inputs)
+
+        # Create data directory and copy files into it
+        os.mkdir(os.path.join(self.data_dir, str(de_obj.id)))
+        shutil.copy(de_file, os.path.join(self.data_dir, str(de_obj.id)))
+
+        # Get JSON for volcano plot
+        with gzip.open(os.path.join(self.test_files_path, '{}.json.gz'.format(de_name)), 'r') as f:
+            de_data = json.load(f)
+
+        # TODO: reference on existing true files
+        de_obj.output = {
+            'diffexp': {'file': de_file},
+            'de_data': de_data,
+            'diffexp': {'file': de_file},
+            'transcript_diff_exp': {'file': de_file},
+            'cds_diff_exp': {'file': de_file},
+            'tss_group_diff_exp': {'file': de_file},
+            'cuffdiff_output': {'file': de_file}
+        }
+
+        de_obj.status = Data.STATUS_DONE
+        de_obj.save()
+
+        self.stdout.write('---- new differential expression ----')
```

Comment:
Use `logger.info` instead of `self.stdout.write`.

https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/management/commands/generate_samples.py#L279